### PR TITLE
Add London and Altair info

### PR DESCRIPTION
**Pull request by timbeiko**
## Description

Fixes https://github.com/ethereum/ethereum-org-website/issues/2918

### CODE REVIEW

**Review by wackerow — APPROVED** [review]
@timbeiko Nice! Thanks for putting this up... (made a couple non-content adjustments)

File: src/content/history/index.md [hunk 1]
```diff
@@ -24,6 +24,32 @@ These rule changes may create a temporary split in the network. New blocks could
...
+
+- [Read the London upgrade specification](https://github.com/ethereum/eth1.0-specs/blob/master/network-upgrades/mainnet-upgrades/london.md)
+
+<ExpandableCard title="London EIPs" contentPreview="Improvements included in this upgrade.">
+
+- [EIP-1559](https://eips.ethereum.org/EIPS/eip-1559) – _improves the transaction fee market_
+- [EIP-3198](https://eips.ethereum.org/EIPS/eip-3198) – _returns the `BASEFEE` from a block_
+- [EIP-3238](https://eips.ethereum.org/EIPS/eip-3238) – _delays the Ice Age_
+
+_Note: more EIPs may be added to the upgrade over the comming weeks, and the EIPs above may still be modified prior to the upgrade being deployed._
```
Comment (ryancreatescopy) [inline src/content/history/index.md:39]: "coming" typo

File: src/content/history/index.md [hunk 2]
```diff
@@ -24,6 +24,32 @@ These rule changes may create a temporary split in the network. New blocks could
...
+
+</ExpandableCard>
+
+---
+
+### (_In Progress_) Altair {#altair}
+
+The Altair upgrade is the first scheduled upgrade for the Beacon Chain. It is expected to go live in 2021. It will add support for "sync committees", which can enable light clients, and will bring inactivity and slashing penalties up to their full values.
```
Comment (ryancreatescopy) [inline src/content/history/index.md:47]: link to beacon chain page if possible `[Beacon Chain](/eth2/beacon-chain)`

**Review by ryancreatescopy — CHANGES_REQUESTED** [review]

File: src/content/history/index.md (hunk 2 above)
Comment (timbeiko) [reply src/content/history/index.md]: Fixed, thanks @ryancreatescopy

File: src/content/history/index.md (hunk 1 above)
Comment (timbeiko) [reply src/content/history/index.md]: Fixed, thanks @ryancreatescopy

**Pull request merged**